engines/xeen/party.cpp

```diff
@@ -661,10 +661,18 @@ int Party::subtract(ConsumableType consumableId, uint amount, PartyBank whereId,
 	return true;
 }
 
+const char *Party::getConsumableForm(ConsumableType consumableId) {
+	switch (consumableId) {
+	case CONS_GOLD: return Res.CONSUMABLE_GOLD_FORMS[0];
+	case CONS_GEMS: return Res.CONSUMABLE_GEM_FORMS[0];
+	}
+	return Res.CONSUMABLE_NAMES[consumableId];
```

Comment:
it would be cleaner to add this in a default case above

Reply:
Fixed